### ISSUE TITLE
Could not see it working

### ISSUE THREAD

**Issue by chechani** [pibico/pibiDAV#1]
We have followed the steps mentioned, NextCloud and ERPNexrt both are on HTTPS, the requirement being installed auto, found settings auto-populated on installation, every setting suggested by you seemed fine, still, w could not see it working and no error log. Updated your app as you just push modification. We could not see working.

Any hint on what we might be missing.

**Comment by pibico** [comment]
Could we make a videoCall and you sharing the screen to see if I can help?

**Comment by chechani** [comment]
We did everything, a screenshot is attached before two hours, we did everything except update requirements but checked that requirement mentioned is satisfied. We will review it again and share the outcome, meantime screenshot is attached. yes we can connect over call too
![WhatsApp Image 2022-04-29 at 3 05 37 PM](https://user-images.githubusercontent.com/6291292/165930893-1bdd59e6-c5f1-42e6-aa87-a96ca9208c22.jpeg)

**Comment by chechani** [comment]
yes, we have set the doctype in settings, below is error we are getting
<img width="1283" alt="Screenshot 2022-04-29 at 16 21 31" src="https://user-images.githubusercontent.com/6291292/165931241-d1b95c63-61bc-446d-a08c-10d41e8a4fd1.png">

**Comment by chechani** [comment]
It seems that NCBrowser could not be imported , we sent a mail for meeting request on your mail

**Comment by chechani** [comment]
Thanks, we can see it working ignoring NC Browser, will evaluate further as we could see it working!
